testdata/repos/standalone/SrcUriCheck/UnstableSrcUri/UnstableSrcUri-0.ebuild

```diff
@@ -0,0 +1,7 @@
+DESCRIPTION="Ebuild with unstable SRC_URI"
+HOMEPAGE="https://github.com/pkgcore/pkgcheck"
+SRC_URI="
+	https://patch-diff.githubusercontent.com/raw/pkgcore/pkgcheck/pull/599.patch
```

Comment:
Will this check catch also the pre-redirect `https://github.com/foo/bar/pull/1234.patch` form? Perhaps we should do that as well.

Reply:
It didn't catch in original version, updated and included in tests.